bug: chunking doesn't work on TSV

`partition_tsv()` ignores the `chunking_strategy` argument. This is because the `@add_chunking_strategy` decorator is missing from the `partition_tsv()` function definition.

**To Reproduce**
```python
def it_chunks_the_partitioned_elements_when_so_instructed():
    elements = partition_tsv(
        "example-docs/stanley-cups.tsv", chunking_strategy="basic", max_characters=50
    )
    assert len(elements) == 10, f"expected 10, got {len(elements)}"

>>> it_chunks_the_partitioned_elements_when_so_instructed()
AssertionError: expected 10, got 1
```

**Additional context**
Even though `partition_tsv()` produces a single `Table` element, chunking should still be available (as it is on the similar `partition_csv()`) because the `Table` element can and often will be larger than the desired chunk size and must be divided into smaller chunks.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,9 +5,6 @@
 * **Remove `page_number` metadata fields** for HTML partition until we have a better strategy to decide page counting.
 * **Extract OCRAgent.get_agent().** Generalize access to the configured OCRAgent instance beyond its use for PDFs.
 * **Add calculation of table related metrics which take into account colspans and rowspans**
-* **Add chunking to partition_tsv** Even though partition_tsv() produces a single Table element,
-  chunking is made available because the Table element is often larger than the desired chunk
-  size and must be divided into smaller chunks.
 
 ### Features
 
@@ -18,6 +15,7 @@
 * **`partition_docx()` handles short table rows.** The DOCX format allows a table row to start late and/or end early, meaning cells at the beginning or end of a row can be omitted. While there are legitimate uses for this capability, using it in practice is relatively rare. However, it can happen unintentionally when adjusting cell borders with the mouse. Accommodate this case and generate accurate `.text` and `.metadata.text_as_html` for these tables.
 * **Remedy macOS test failure not triggered by CI.** Generalize temp-file detection beyond hard-coded Linux-specific prefix.
 * **Remove unnecessary warning log for using default layout model.**
+* **Add chunking to partition_tsv** Even though partition_tsv() produces a single Table element, chunking is made available because the Table element is often larger than the desired chunk size and must be divided into smaller chunks.
 
 ## 0.13.6
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,10 +1,13 @@
-## 0.13.7-dev8
+## 0.13.7-dev9
 
 ### Enhancements
 
 * **Remove `page_number` metadata fields** for HTML partition until we have a better strategy to decide page counting.
 * **Extract OCRAgent.get_agent().** Generalize access to the configured OCRAgent instance beyond its use for PDFs.
 * **Add calculation of table related metrics which take into account colspans and rowspans**
+* **Add chunking to partition_tsv** Even though partition_tsv() produces a single Table element,
+  chunking is made available because the Table element is often larger than the desired chunk
+  size and must be divided into smaller chunks.
 
 ### Features
 

--- a/test_unstructured/partition/csv/test_tsv.py
+++ b/test_unstructured/partition/csv/test_tsv.py
@@ -240,7 +240,9 @@ def test_partition_csv_header():
 
 
 def test_add_chunking_strategy_to_partition_tsv_non_default():
+    """The same chunks are returned if chunking elements or chunking during partitioning."""
     elements = partition_tsv(filename=example_doc_path("stanley-cups.tsv"))
+    chunks = chunk_by_title(elements, max_characters=9, combine_text_under_n_chars=0)
 
     chunk_elements = partition_tsv(
         example_doc_path("stanley-cups.tsv"),
@@ -249,7 +251,5 @@ def test_add_chunking_strategy_to_partition_tsv_non_default():
         combine_text_under_n_chars=0,
         include_header=False,
     )
-    chunks = chunk_by_title(elements, max_characters=9, combine_text_under_n_chars=0)
 
-    assert chunk_elements != elements
     assert chunk_elements == chunks

--- a/test_unstructured/partition/csv/test_tsv.py
+++ b/test_unstructured/partition/csv/test_tsv.py
@@ -229,7 +229,7 @@ def test_partition_tsv_element_metadata_has_languages():
     assert elements[0].metadata.languages == ["eng"]
 
 
-def test_partition_csv_header():
+def test_partition_tsv_header():
     filename = "example-docs/stanley-cups.tsv"
     elements = partition_tsv(filename=filename, strategy="fast", include_header=True)
     assert (
@@ -239,8 +239,8 @@ def test_partition_csv_header():
     assert "<thead>" in elements[0].metadata.text_as_html
 
 
-def test_add_chunking_strategy_to_partition_tsv_non_default():
-    """The same chunks are returned if chunking elements or chunking during partitioning."""
+def test_partition_tsv_supports_chunking_strategy_while_partitioning():
+    # The same chunks are returned if chunking elements or chunking during partitioning.
     elements = partition_tsv(filename=example_doc_path("stanley-cups.tsv"))
     chunks = chunk_by_title(elements, max_characters=9, combine_text_under_n_chars=0)
 

--- a/test_unstructured/partition/csv/test_tsv.py
+++ b/test_unstructured/partition/csv/test_tsv.py
@@ -240,7 +240,6 @@ def test_partition_tsv_header():
 
 
 def test_partition_tsv_supports_chunking_strategy_while_partitioning():
-    # The same chunks are returned if chunking elements or chunking during partitioning.
     elements = partition_tsv(filename=example_doc_path("stanley-cups.tsv"))
     chunks = chunk_by_title(elements, max_characters=9, combine_text_under_n_chars=0)
 
@@ -252,4 +251,5 @@ def test_partition_tsv_supports_chunking_strategy_while_partitioning():
         include_header=False,
     )
 
+    # The same chunks are returned if chunking elements or chunking during partitioning.
     assert chunk_elements == chunks

--- a/test_unstructured/partition/csv/test_tsv.py
+++ b/test_unstructured/partition/csv/test_tsv.py
@@ -10,6 +10,7 @@ from test_unstructured.partition.test_constants import (
     EXPECTED_TEXT_XLSX,
 )
 from test_unstructured.unit_utils import assert_round_trips_through_JSON, example_doc_path
+from unstructured.chunking.title import chunk_by_title
 from unstructured.cleaners.core import clean_extra_whitespace
 from unstructured.documents.elements import Table
 from unstructured.partition.tsv import partition_tsv
@@ -236,3 +237,19 @@ def test_partition_csv_header():
         == "Stanley Cups Unnamed: 1 Unnamed: 2 " + EXPECTED_TEXT_XLSX
     )
     assert "<thead>" in elements[0].metadata.text_as_html
+
+
+def test_add_chunking_strategy_to_partition_tsv_non_default():
+    elements = partition_tsv(filename=example_doc_path("stanley-cups.tsv"))
+
+    chunk_elements = partition_tsv(
+        example_doc_path("stanley-cups.tsv"),
+        chunking_strategy="by_title",
+        max_characters=9,
+        combine_text_under_n_chars=0,
+        include_header=False,
+    )
+    chunks = chunk_by_title(elements, max_characters=9, combine_text_under_n_chars=0)
+
+    assert chunk_elements != elements
+    assert chunk_elements == chunks

--- a/unstructured/__version__.py
+++ b/unstructured/__version__.py
@@ -1,1 +1,1 @@
-__version__ = "0.13.7-dev8"  # pragma: no cover
+__version__ = "0.13.7-dev9"  # pragma: no cover

--- a/unstructured/partition/tsv.py
+++ b/unstructured/partition/tsv.py
@@ -5,6 +5,7 @@ from typing import IO, Any, Optional
 import pandas as pd
 from lxml.html.soupparser import fromstring as soupparser_fromstring
 
+from unstructured.chunking import add_chunking_strategy
 from unstructured.documents.elements import (
     Element,
     ElementMetadata,
@@ -25,6 +26,7 @@ DETECTION_ORIGIN: str = "tsv"
 
 @process_metadata()
 @add_metadata_with_filetype(FileType.TSV)
+@add_chunking_strategy
 def partition_tsv(
     filename: Optional[str] = None,
     file: Optional[IO[bytes]] = None,

